src/app/modules/main/module.nim

```diff
@@ -1231,9 +1234,36 @@ method onAcceptRequestToJoinSuccess*[T](self: Module[T], communityId: string, me
     item.updatePendingRequestLoadingState(memberKey, false)
 
 method onMembershipStatusUpdated*[T](self: Module[T], communityId: string, memberPubkey: string, status: MembershipRequestState) =
-  let item = self.view.model().getItemById(communityId)
-  if item.id != "":
-    item.updateMembershipStatus(memberPubkey, status)
+  let myPublicKey = singletonInstance.userProfile.getPubKey()
+  let communityDto = self.controller.getCommunityById(communityId)
+
+  if myPublicKey == memberPubkey:
+    case status:
+      of MembershipRequestState.Banned:
+        singletonInstance.globalEvents.showCommunityMemberBannedNotification(fmt "You've been banned from {communityDto.name}", "", communityId)
+      of MembershipRequestState.Kicked:
+        singletonInstance.globalEvents.showCommunityMemberKickedNotification(fmt "You were kicked from {communityDto.name}", "", communityId)
+      of MembershipRequestState.Unbanned:
+        singletonInstance.globalEvents.showCommunityMemberUnbannedNotification(fmt "You were unbanned from {communityDto.name}", "", communityId)
+      else:
+        discard
+  else:
```

Comment:
Will this show a notification to other members too or just the owner? I feel like normal users don't need to see this

Reply:
`else` will be visible only to the user, who clicked kick/ban/unban
to point it out, I added a check that such notification must be shown for a control node only (previously it was shown for admins and TM too, but taking into account that they need to wait for a confirmation from the owner side, I disabled it)